packages/cli/package.json

```diff
@@ -33,6 +33,7 @@
   },
   "scripts": {
     "test": "NODE_OPTIONS=--experimental-vm-modules npx jest --silent --coverage --runInBand --forceExit",
+    "postinstall": "shx cp -r ../../node_modules/multiaddr node_modules/multiaddr && patch-package",
```

Comment:
There are 2 other packages that use the outdated IPFS HTTP client - `ipfs-daemon` and `pinning-ipfs-backend`. IMO, only the Ceramic CLI has the crash vulnerability, but I can also patch the others to be safe. WDYT?

Reply:
I guess if you test with the bundled ipfs and do some pinning/unpinning requests and it all works then it's probably fine?